### PR TITLE
LPS-186796 Remove Optional usages in module: portal-search-web-api final

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:search

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/search/bar/portlet/helper/SearchBarPrecedenceHelper.java [hunk 1]
```diff
@@ -138,13 +137,11 @@ private SearchBarPortletPreferences _getSearchBarPortletPreferences(
 		Portlet portlet, ThemeDisplay themeDisplay) {
 
 		if (portlet == null) {
-			return new SearchBarPortletPreferencesImpl(Optional.empty());
+			return new SearchBarPortletPreferencesImpl(null);
 		}
```
Comment (julschong) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/search/bar/portlet/helper/SearchBarPrecedenceHelper.java:141]: @tinatian portlet null check is added back

**Comment by tinatian** [comment]
ci:forward:force

**Comment by julschong** [comment]
ci:forward

**Comment by julschong** [comment]
see https://github.com/brianchandotcom/liferay-portal/pull/136569

**Pull request closed without merging**